app/index.js

```diff
@@ -34,12 +34,15 @@ module.exports = yeoman.generators.Base.extend({
       defaults: 8080
     });
 
-    this.option('test-framework', {
+    this.option('frameworks', {
       type: String,
-      desc: 'Specifies which testing framework to use',
+      desc: 'Specifies which testing frameworks to use (comma separated)',
       defaults: 'jasmine'
     });
-    this.options['test-framework'] = this.options['test-framework'].toLowerCase();
+    this.options.frameworks = notEmpty(this.options.frameworks);
```

Comment:
I believe this consistent with the way that the other comma separated options are read - using `notEmpty()` which takes a string and returns an array, right?